### ISSUE TITLE
Stanford University's iOS course 

### ISSUE THREAD

**Issue by swatisingh45** [facebook-developer-circle-delhi/Community-Curated-Resources#54]
Youtube link to the Stanford University's iOS course is also present, besides being on iTunes. Shall I add that as well?

**Comment by nsnsns** [comment]
Yes please do add this course.
Btw where is it present? In YouTube? Or where?

**Comment by saranshkataria** [comment]
@swatisingh45 is the iTunes one less accessible on some platforms or something like that?

**Comment by nsnsns** [comment]
@saranshkataria @swatisingh45 yes the iTunes one is less accessible as people with android devices cannot access the iTunes tutorial -- therefore for the sake of group members using Android devices (= 95% of group members) who want to see tutorial on the go (say in metro) -- I request to please put the iOS tutorial on the group

**Comment by swatisingh45** [comment]
@saranshkataria, @nsnsns is totally correct. Shall add the link below the iTunes link and submit a PR?
